Dockerfile

```diff
@@ -52,6 +52,23 @@ COPY . /work
 WORKDIR /work
 RUN ./pkg/deb/build.sh
 
+FROM ubuntu:hirsute AS hirsute
+
+RUN set -x; apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y -t hirsute-backports install debhelper && \
```

Comment:
You actually shouldn't need to explicitly install `debhelper` either — it's going to be pulled in via `cdbs` [1]. We only had to do this on Xenial because the automatically installed version of `debhelper` was too old. Notice that we don't do this on either Bionic or Focal.

[1] [cdbs](https://packages.ubuntu.com/hirsute/cdbs) -> [dh-translations](https://packages.ubuntu.com/hirsute/dh-translations) -> [debhelper](https://packages.ubuntu.com/hirsute/debhelper)

Reply:
Updated.